packages/kbn-securitysolution-hook-utils/.babelrc

```
@@ -0,0 +1,3 @@
+{
+  "presets": ["@kbn/babel-preset/node_preset"]
```

```suggestion
  "presets": ["@kbn/babel-preset/node_preset"],
  "ignore": ["**/*.test.ts", "**/*.test.tsx"]  
```

You want ignore here like the rest?